### PR TITLE
Bad image repository name

### PR DESCRIPTION
**Pull request by darioAnongba**
Bad image repository name

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -67,6 +64,8 @@ services:
     command: run
     volumes:
       - ${HOST_MANIFEST_PATH-./deps/config/manifests}:/manifests:ro
+    restart: "on-failure"
+    tty: true
```
Comment (dolanor) [inline docker-compose.yml:68]: Why the tty?

File: docker-compose.yml [hunk 2]
```diff
@@ -44,21 +37,25 @@ services:
...
 
   migration:
-    <<: *container-common
+    image: consensys/quorum-key-manager:latest
     environment:
       <<: *default-variables
     command: migrate up
-    restart: "no"
+    restart: "on-failure"
+    tty: true
```
Comment (dolanor) [inline docker-compose.yml:49]: Why the tty

File: docker-compose.yml [hunk 3]
```diff
@@ -44,21 +37,25 @@ services:
       interval: 1s
       timeout: 1s
       retries: 5
+    restart: "on-failure"
+    tty: true
```
Comment (dolanor) [inline docker-compose.yml:41]: why the tty?

File: docker-compose.yml (hunk 1 above)
Comment (darioAnongba) [reply docker-compose.yml]: removed

**Pull request merged**